src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: `${name}`,
+    wheels: `${wheels}`,
+    version: `${version}`,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
```

Comment:
```suggestion
    coords: {
      x: 0,
      y: 0,
    },

    get info() {
      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
    },
```
add an empty row between methods to increase readability